Terminal.Gui/Views/TextField.cs

```diff
@@ -1151,7 +1183,7 @@ void PrepareSelection (int x, int direction = 0)
 		/// </summary>
 		public void ClearAllSelection ()
 		{
-			if (_selectedStart == -1 && _length == 0 && _selectedText == "")
+			if (_selectedStart == -1 && _length == 0 && string.IsNullOrEmpty (_selectedText))
```

Comment:
Braces.

Reply:
Fixed with the https://github.com/gui-cs/Terminal.Gui/pull/2682/commits/0b39b80fc8dcb7658fbe6e2d24933b3df2b80784.